spring-kafka/src/main/java/org/springframework/kafka/listener/ListenerUtils.java

```diff
@@ -144,6 +139,17 @@ public static void unrecoverableBackOff(BackOff backOff, Map<Thread, BackOffExec
 		}
 	}
 
+	public static long nextBackOff(BackOff backOff, Map<Thread, BackOffExecution> executions) {
```

Comment:
Not sure if this method supposed to be `public`, but if it is, then it has to have respective Javadoc.